spec/xcpretty/parser_spec.rb

```diff
@@ -225,6 +225,17 @@ module XCPretty
       @parser.parse(SAMPLE_TOUCH)
     end
 
+    it "parses write file" do
+      @formatter.should receive(:format_write_file).with(
+      '/Users/me/myproject/Build/Intermediates/Pods.build/Debug-iphonesimulator/Pods-AFNetworking.build/Objects-normal/x86_64/Pods-AFNetworking.LinkFileList')
+      @parser.parse(SAMPLE_WRITE_FILE)
+    end
+
+    it "parses write auxiliary files" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.